### ISSUE TITLE
"release" GitHub action has been failing since June 28th, 2022

### ISSUE THREAD

**Issue by DorianListens** [rust-lang/rust-analyzer#13062]
Obviously r-a has been releasing successfully, but the "publish" job of the "release" GitHub action has been failing for the last couple of months.

The last successful run was https://github.com/rust-lang/rust-analyzer/actions/runs/2572925904
The first failing run was https://github.com/rust-lang/rust-analyzer/actions/runs/2579927687

The failures appear consistent, caused by an external server error encountered by the `npx osvx publish` step.

**Comment by Veykril** [comment]
This is known, we set up everything for openvsx publishing but its still failing on their side hence the release "failure" https://github.com/rust-lang/rust-analyzer/issues/11080

**Comment by lnicola** [comment]
Well, `publish.yml` also fails, but I'm not sure how to fix it:

```
  feature `in-rust-tree` includes `sourcegen/in-rust-tree`, but `sourcegen` is not a dependency
```

**Comment by Veykril** [comment]
Well from the looks of it, sourcegen doesn't even need the feature? It's source doesn't make use of it at least. Aside from that I don't think cargo gives you tools for managing dev-depency features like that

**Comment by lnicola** [comment]
Fixed in #13295.
